stetho-sample/src/debug/java/com/facebook/stetho/sample/SampleDebugApplication.java

```diff
@@ -42,7 +49,56 @@ private void initializeStetho(final Context context) {
                 .finish();
           }
         })
-        .enableWebKitInspector(Stetho.defaultInspectorModulesProvider(context))
+        .enableWebKitInspector(new ExtInspectorModulesProvider(context))
         .build());
   }
+
+  private static class ExtInspectorModulesProvider implements InspectorModulesProvider {
+
+    private Context mContext;
+
+    ExtInspectorModulesProvider(Context context) {
+      mContext = context;
+    }
+
+    @Override
+    public Iterable<ChromeDevtoolsDomain> get() {
+      return new Stetho.DefaultInspectorModulesBuilder(mContext)
+          .provideDatabaseDriver(createContentProviderDatabaseDriver(mContext))
+          .finish();
+    }
+
+    @TargetApi(Build.VERSION_CODES.ICE_CREAM_SANDWICH)
+    private ContentProviderDatabaseDriver createContentProviderDatabaseDriver(Context context) {
+      ContentProviderSchema calendarsSchema = new ContentProviderSchema.Builder()
+          .table(new Table.Builder()
+              .uri(CalendarContract.Calendars.CONTENT_URI)
+              .projection(new String[] {
+                  CalendarContract.Calendars._ID,
+                  CalendarContract.Calendars.NAME,
+                  CalendarContract.Calendars.ACCOUNT_NAME,
+                  CalendarContract.Calendars.IS_PRIMARY,
+              })
+              .build())
+          .build();
+
+      // sample events content provider we want to support
+      ContentProviderSchema eventsSchema = new ContentProviderSchema.Builder()
+          .table(new Table.Builder()
+              .uri(CalendarContract.Events.CONTENT_URI)
+              .projection(new String[]{
```

Comment:
Yes, it's right. The issue, is with a lot of selected columns. For example in android content provider of `event ` there is like more than 20 columns. The problem is that the view of table in chrome makes everything shrinked due to luck of space and it's impossible to hide or resize columns in such case. And also, not all columns have interesting info. Beside this, I have no other better explanation of selecting only few columns in this sample, so using  `.defaultProjection()` is also fine :)